### ISSUE TITLE
Add validation for zero sized file attachments

### ISSUE THREAD

**Issue by mattcdavis1** [verbb/formie#348]
**issue**
Currently, there is validation for maximum file size on uploads but no option to check for zero sized file attachments. The reason this is needed is because zero sized file size attachments can cause SMTP email sending issues. [Here is a screenshot](https://www.screencast.com/t/aiZistQU34) from Postmark App where a notification bounced due to a zero sized file attachments.

**proposed solution**
Add a validation option for file uploads (ideally enabled by default) that checks that an uploaded attachment does not have a zero file size.

**Comment by engram-design** [comment]
Should be fixed for the next release. I've added a "Min File Size" setting to file upload fields and am also rejecting any zero-sized file uploads so they don't get attached to emails in the first place.

To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.17",
  "...": "..."
}
```

Then run `composer update`.

**Comment by mattcdavis1** [comment]
great - thanks!

**Comment by engram-design** [comment]
Fixed in [1.3.18](https://github.com/verbb/formie/releases/tag/1.3.18)
